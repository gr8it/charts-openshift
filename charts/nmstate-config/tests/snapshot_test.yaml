suite: Snapshot test
templates:
  - templates/*
values:
  - ../values.lint.yaml
release:
  name: nmstate-config
  namespace: openshift-nmstate

tests:
  - it: manifests should match snapshot
    asserts:
      - matchSnapshot: {}
