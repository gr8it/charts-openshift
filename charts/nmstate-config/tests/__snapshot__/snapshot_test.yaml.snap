manifests should match snapshot:
  1: |
    apiVersion: nmstate.io/v1
    kind: NMState
    metadata:
      annotations:
        argocd.argoproj.io/sync-wave: "-40"
      name: nmstate
  2: |
    apiVersion: nmstate.io/v1
    kind: NodeNetworkConfigurationPolicy
    metadata:
      annotations:
        argocd.argoproj.io/sync-wave: "-20"
      name: bond1
    spec:
      desiredState:
        interfaces:
          - ipv4:
              dhcp: false
              enabled: false
            ipv6:
              dhcp: false
              enabled: false
            link-aggregation:
              mode: 802.3ad
              options:
                miimon: "140"
                xmit_hash_policy: layer2
              port:
                - ens13f1np1
                - ens7f1np1
            name: bond1
            state: up
            type: bond
      nodeSelector:
        node-role.kubernetes.io/worker: ""
